assets/src/edit-story/theme.js

```diff
@@ -19,19 +19,24 @@
  */
 import { createGlobalStyle, ThemeContext } from 'styled-components';
 import { rgba } from 'polished';
+import deepmerge from 'deepmerge';
```

Comment:
We have this through some other dependency, so it builds, but we don't directly depend on it, so it doesn't lint.

Reply:
It would need to be added to `package.json` explicitly

Reply:
It's temporary only until #6209 is merged in, that's why I didn't add it. I just needed it to test locally.

Reply:
Now removed again.